crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -0,0 +1,213 @@
+        #[rust_name = "set_capitalization"]
+        fn setCapitalization(self: &mut QFont, caps: QFontCapitalization);
+
+        /// Sets the family name of the font. The name is case insensitive and may include a foundry name.
+        #[rust_name = "set_family"]
+        fn setFamily(self: &mut QFont, family: &QString);
+
+        /// If enable is true, sets fixed pitch on; otherwise sets fixed pitch off.
+        #[rust_name = "set_fixed_pitch"]
+        fn setFixedPitch(self: &mut QFont, enable: bool);
+
+        /// Set the preference for the hinting level of the glyphs to hintingPreference.
+        #[rust_name = "set_hinting_preference"]
+        fn setHintingPreference(self: &mut QFont, hintingPreference: QFontHintingPreference);
+
+        /// Sets the style() of the font to QFont::StyleItalic if enable is true; otherwise the style is set to QFont::StyleNormal.
+        #[rust_name = "set_italic"]
+        fn setItalic(self: &mut QFont, enable: bool);
+
+        /// Enables kerning for this font if enable is true; otherwise disables it. By default, kerning is enabled.
+        #[rust_name = "set_kerning"]
+        fn setKerning(self: &mut QFont, enable: bool);
+
+        /// Sets the letter spacing for the font to spacing and the type of spacing to type.
+        #[rust_name = "set_letter_spacing"]
+        fn setLetterSpacing(self: &mut QFont, spacingType: QFontSpacingType, spacing: f64);
+
+        /// If enable is true, sets strikeout on; otherwise sets strikeout off.
+        #[rust_name = "set_strikeout"]
+        fn setStrikeOut(self: &mut QFont, enable: bool);
+
+        /// Sets the style of the font to style.
+        #[rust_name = "set_style"]
+        fn setStyle(self: &mut QFont, style: QFontStyle);
+
+        /// Sets the style name of the font to styleName.
+        #[rust_name = "set_style_name"]
+        fn setStyleName(self: &mut QFont, styleName: &QString);
+
+        /// Returns true if strikeout has been set; otherwise returns false.
+        fn strikeOut(self: &QFont) -> bool;
+
+        /// If enable is true, sets underline on; otherwise sets underline off.
+        #[rust_name = "set_underline"]
+        fn setUnderline(self: &mut QFont, enable: bool);
+
+        /// Returns true if underline has been set; otherwise returns false.
+        fn underline(self: &QFont) -> bool;
+    }
+
+    #[namespace = "rust::cxxqtlib1"]
+    unsafe extern "C++" {
+        include!("cxx-qt-lib/common.h");
+        type QFontStyle;
+        type QFontHintingPreference;
+        type QFontCapitalization;
+        type QFontSpacingType;
+
+        #[doc(hidden)]
+        #[rust_name = "qfont_init_default"]
+        fn construct() -> QFont;
+
+        #[doc(hidden)]
+        #[rust_name = "qfont_drop"]
+        fn drop(pen: &mut QFont);
+    }
+}
+
+#[derive(Clone)]
+#[repr(C)]
+pub struct QFont {
+    _cspec: MaybeUninit<i32>,
+    _resolve_mask: MaybeUninit<u16>,
+}
+
```

Comment:
clone ?

Reply:
As in i meant you need to add a `impl Clone for QFont` because of the static assert on the C++ side
```cpp
static_assert(!::std::is_trivially_copy_constructible<QFont>::value);
```

Reply:
(you can't use `derive(Clone)` because it's not trivial)

Reply:
oops ! Ah ok ! :) 
I will do it